specification/cognitiveservices/AnomalyDetector/multivariate/routes.tsp

```diff
@@ -57,15 +58,19 @@ op trainMultivariateModel is MultivariateServiceAction<
   // TParams
   {
     @doc("Model information.")
-    @body modelInfo: ModelInfo,
+    @body
+    modelInfo: ModelInfo;
   },
   // TResponse
   {
-    @statusCode statusCode: 201,
+    @statusCode statusCode: 201;
+
     @doc("Location and ID of the model.")
-    @header location: string,
-    @body result: AnomalyDetectionModel
-  }>;
+    @header
+    location: string;
+    @body result: AnomalyDetectionModel;
```

Comment:
@timotheeguerin: Why is `@header` split onto two lines but `@body` stays on one line?

Reply:
one decorator can be inlined(it should keep your decision), more than one it will split lines